front/src/styles/scss/_variables.scss

```diff
@@ -12,4 +12,9 @@
   --coolgray5: #b9b9b9;
   --coolgray3: #d7d7d7;
   --coolgray1: #f2f2f2;
+  --warning-orange: #ffb612; // from text-warning bootstrap class
+  --error-red: red;
+  --success-green: green;
+  --modal-gray-transparent-background: rgba(0,0,0,0.1);
+  --card-background-white: white;
```

Comment:
I'm sorry, we must have misunderstood each other, no need to create new variables they already exist!

Please use them :)

![image](https://user-images.githubusercontent.com/6003856/214843528-c1890d6f-8424-40ea-bef9-575414119522.png)